Makefile

```diff
@@ -4,12 +4,57 @@ ifdef JENKINS_URL
 	MAKE_INCLUDES := .
 else
 	# For local dev, use the installed vivarium_build_utils package
-	MAKE_INCLUDES := $(shell python -c "from vivarium_build_utils.resources import get_makefiles_path; print(get_makefiles_path())")
+	MAKE_INCLUDES := $(shell python -c "import vivarium_build_utils.resources" &>/dev/null && python -c "from vivarium_build_utils.resources import get_makefiles_path; print(get_makefiles_path())")
 endif
 
-PACKAGE_NAME = vivarium
+PACKAGE_NAME = $(notdir $(CURDIR))
 
 # Include makefiles from vivarium_build_utils
+ifneq ($(MAKE_INCLUDES),) # if not an empty string
 include $(MAKE_INCLUDES)/base.mk
 include $(MAKE_INCLUDES)/test.mk
+endif
+
+ifeq ($(MAKE_INCLUDES),) # empty string
```

Comment:
Can't this be an `else` in the above `if` block?

Reply:
whoops, yeah I misread the comment - I'll do this